test/resources/controls/aws_parallelcluster_install/package_repos_spec.rb

```diff
@@ -0,0 +1,43 @@
+# Use the name matching the resource type
+control 'package_repos' do
```

Comment:
can we remove the `_spec` suffix from the file name or is it there by purpose?

Reply:
It is the convention for inspec